docs/LV_client/shared_libraries/mono_lane_follower_lights.md

```diff
@@ -0,0 +1,34 @@
+# mono_lane_follower_lights.vi
+
+<p class="img_container">
+<img class="lg_img" src="../mono_lane_follower_lights.png"/>
+</p>
+
+### Description
+
+This example shows how to use the monoDrive client to use a a compiled DLL to create a lane follower controller. This example allows the user to configure and update the lights attached to the ego vehicle. 
+
```

Comment:
Consider this wording instead which removes an additional "a" and moves the last part of the sentence to the middle: "This example shows how to use the monoDrive client to  create a lane follower controller using a compiled DLL. This example allows the user to configure and update the lights attached to the ego vehicle."